### ISSUE TITLE
Accessibility of E-Mail Input Fields

### ISSUE THREAD

**Issue by jung-tim** [freescout-help-desk/freescout#4869]
The input/search fields used to create or search for ticket recipients are not accessible to screen reader users.
For this reason, blind and visually impaired users can only create a ticket with great difficulty.

**Comment by freescout-help** [comment]
Can you send screenshots.

**Comment by jung-tim** [comment]
No, but I can explain the challenge further.
If a screen reader user opens the corresponding field and starts typing an email address or a name, no search results are displayed that are readable by the screen reader.
Likewise, data entered into the field cannot be deleted because the delete button is not accessible. Therefore, it is also not possible to take screenshots of this, as it is an accessibility issue that is not reflected on the screen.

**Comment by freescout-help** [comment]
> The input/search fields used to create or search for ticket recipients

Where are you doing this? On which page? Please write a step-by-step instruction.

**Comment by philippkraft** [comment]
Just a comment from my screen reader using colleague: she has the issue too (everywhere), but it is a common problem with screen readers. Even Google's suggestion list is not working.
